### PR TITLE
fix(#8494): update lodash

### PR DESCRIPTION
**Pull request by garethbowen**
# Description

[description]

medic/cht-core#8494

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update-lodash/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update-lodash/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:update-lodash/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: admin/src/js/services/search.js [hunk 1]
```diff
@@ -134,55 +58,9 @@ const Search = require('@medic/search');
...
-                });
-
-                if (relevantDataRecord) {
-                  Object.assign(relevantDataRecord, dateResult.value);
-                  relevantDataRecord.sortByLastVisitedDate = extensions.sortByLastVisitedDate;
-                }
-              });
-
-              return r.dataRecords;
-            });
```
Comment (garethbowen) [inline admin/src/js/services/search.js:185]: All these options were not used in the admin app.

File: admin/src/js/services/search.js [hunk 2]
```diff
@@ -1,5 +1,4 @@
-const _ = require('lodash/core');
-const moment = require('moment');
+const _ = require('lodash'); // #8494 don't use eslint/core as it throws an exception
```
Comment (garethbowen) [inline admin/src/js/services/search.js:1]: This was the fix. The bug only impacts the "core" bundle. The downside is that the bundle gets bigger, but not by much.

**Comment by garethbowen** [comment]
Review please @latin-panda . I took the time to remove a bunch of dead code as well..

**Review by latin-panda — APPROVED** [review]
Happy to see so many deletions!

**Pull request merged**